libs/src/sdk/api/HealthCheckResponseData.ts

```diff
@@ -0,0 +1,135 @@
+export type HealthCheckResponseData = Partial<{
```

Comment:
how was this made? just worried about drift (if we ever deprecate some fields)

Reply:
Manually unf. Would like to autogenerate but this is at least better than `any`. I made it a partial but I could do a deep partial instead to ensure folks check existence. (Also not exporting at root, meant to be internal to libs/sdk)